src/System.Diagnostics.Process/ref/4.0/System.Diagnostics.Process.Manual.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
How should I think about the files in src/System.Diagnostics.Process/ref vs those in src/System.Diagnostics.Process/ref/4.0 ?